### PR TITLE
Add whatsnew_71.lang

### PR DESCRIPTION
**Pull request by alexgibson**
Bedrock PR: https://github.com/mozilla/bedrock/pull/8159
Langchecker PR: https://github.com/mozilla-l10n/langchecker/pull/944

### CODE REVIEW

File: en-US/firefox/whatsnew_71.lang [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+;Send huge files to anyone you want, with self-destructing links.
+Send huge files to anyone you want, with self-destructing links.
+
+
+;Get a Firefox Account
+Get a Firefox Account
+
+
+;Read the <a href="%(notes)s">Release Notes</a> to know more about what’s new in your Firefox Browser.
```
Comment (peiying2) [inline en-US/firefox/whatsnew_71.lang]: Let's add a comment here on the capital B in Firefox Browser. Browser should be preferably left in English but can be translated due to linguistic necessity.

**Comment by peiying2** [comment]
@alexgibson I remember discussion on using firefox/whatsnew.lang (Fx66) page and manually add two strings for key markets for Fx71. Did marketing change their mind? We are only days away from the locale list due date.

**Comment by alexgibson** [comment]
@peiying2 as far as I know the two strings have already been added to that page here: https://github.com/mozilla-l10n/www.mozilla.org/pull/345

**Comment by peiying2** [comment]
Got it. Yes, I am tracking the completion of the old one for releng.

**Comment by alexgibson** [comment]
@peiying2 updated. Please see my update in the langchecker PR. If all looks good and makes sense, I'll extract strings for other locales and merge.

https://github.com/mozilla-l10n/langchecker/pull/944

**Comment by alexgibson** [comment]
Pushed to prod in https://github.com/mozilla-l10n/bedrock-l10n/commit/62f01d1f717859a78396e578aebe219e88271bf5

**Pull request merged**